templates/eap-cd-amq-persistent-s2i.json

```diff
@@ -6,10 +6,10 @@
             "iconClass": "icon-eap",
             "tags": "eap,javaee,java,jboss",
             "version": "14.0",
-            "openshift.io/display-name": "JBoss EAP CD + AMQ 7 (with https)",
+            "openshift.io/display-name": "JBoss EAP CD + AMQ 7 with https (Passthrough TLS)",
```

Comment:
I think you can remove `with https` as it seems redundant.  It also shortens the name.

Reply:
+1. "with passthrough TLS" is good.  No capital P, no parens.